### DISCUSSION TITLE
Unclear how Dividend class is meant to be used

### DISCUSSION THREAD

**Discussion by rivettp** [edmcouncil/fibo#1379] [Securities (SEC)]
(Really need an example)
What is the Commitment then? If in 2019 a Board declared quarterly dividends of $1, $2, none and $1 then are there 4 Dividend instances? Are they created only once a Board has made its determination that quarter? Is there a Schedule at all? How do we capture generally that they are committed to announce a Dividend quarterly (even if it is zero)?

_Originally posted by @rivettp in https://github.com/edmcouncil/fibo/pull/1130#discussion_r480398414_

**Comment by ElisaKendall** [comment]
We need to augment the ontology with some explanatory notes - typically for common shares, each dividend is approved by a board of directors, not scheduled per se, whereas there is a fixed schedule of dividends associated with preferred shares that is set at issuance and documented in the prospectus.  Of course, if business circumstances change, a board can suspend payment of those scheduled dividends, but the schedule is typically set up front.

SO -- to address this issue we should clarify this in notes in the ontology and likely need to develop some examples.
